paramiko/sftp_file.py

```diff
@@ -485,6 +492,13 @@ def readv(self, chunks):
         :param chunks:
             a list of ``(offset, length)`` tuples indicating which sections of
             the file to read
+        :param int max_concurrent_prefetch_requests:
+            The maximum number of concurrent read requests to prefetch.
```

Comment:
I'd replace this param description everywhere but `SFTPClient.get`, with something like:

```
The maximum number of concurrent read requests to prefetch. See the API docs for `.SFTPClient.get` for details.
```

(IOW I don't think it's useful to have the same paragraph in all 4+ places :))